src/plugins/userIndicator/components/UserIndicator.vue

```diff
@@ -94,7 +94,7 @@ export default {
         selectionOptions,
         iconClass: 'alert',
         title: 'Select Role',
-        message: 'Please select your role for operator status.',
+        message: '',
```

Comment:
@evenstensberg would you mind adding a unit, e2e, or visual test to cover this area for regressions?

Reply:
I wish, but I can't reproduce in the UI as I don't know how to get to the user indicator UI. You could push to this branch though

Reply:
@evenstensberg Check out the tests in [this suite](https://github.com/nasa/openmct/blob/master/e2e/tests/functional/plugins/operatorStatus/operatorStatus.e2e.spec.js) for some examples.

Reply:
There's a test suit within the folder the component was created. Adding a test is easy, but I can't find any reproduction step to get to the role view

Reply:
I'll take a look today and update in this thread with instructions

Reply:
@evenstensberg 

First, install the `ExampleUser` and `OperatorStatus` plugins by adding the following to `index.html`:

```html
openmct.install(openmct.plugins.OperatorStatus());
openmct.install(openmct.plugins.example.ExampleUser());
```

Then you should be able to access the role selection dialog via the button in the indicator bar:


https://github.com/nasa/openmct/assets/9259993/ffcc89fc-682a-43dd-a1d3-e08fa8226583

Reply:
Thanks @ozyx ! PTAL